.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -7,26 +7,31 @@ body:
       value: |
         Thank you for contributing to the Klaviyo React Native SDK!
 
-        Before you submit your issue, please complete each text area below with the relevant details for your bug, and complete the steps in the checklist.
+        Before you submit your issue, please read our [contributing guidelines](./CONTRIBUTING.md) and complete each
+        text area below with the relevant details for your bug.
 
-        We welcome your input! If you have any code suggestions regarding your issue, feel free to [submit a pull request](https://github.com/klaviyo/klaviyo-react-native-sdk/pulls) after creating an issue.
+        Note: incomplete issues may be de-prioritized or closed.
+
+        We welcome your input! If you have any code suggestions regarding your issue, feel free to
+        [submit a pull request](https://github.com/klaviyo/klaviyo-react-native-sdk/pulls) after creating an issue.
   - type: textarea
     attributes:
       label: Description
       render: markdown
       description: |
         A short description of the incorrect behavior.
 
-        If you think this issue has been recently introduced and did not occur in an earlier version, please note that. If possible, include the last version that the behavior was correct in addition to your current version.
+        If you think this issue has been recently introduced and did not occur in an earlier version, please note that.
+        If possible, include the last version that the behavior was correct in addition to your current version.
     validations:
       required: true
   - type: checkboxes
     attributes:
       label: Checklist
       options:
-        - label: I have determined whether this bug is also reproducible in a vanilla project
+        - label: I have determined whether this bug is also reproducible in a vanilla project, such as the example app in this repository.
```

Comment:
would it be overkill to add a checkbox for, i've read and understand the contributing policy or something to that effect? can be added to bugs and feature templates

Reply:
Great idea